Cargo.toml

```
@@ -8,6 +8,7 @@ license = "AGPL-3.0-only"
 readme = "README.md"
 
 [dependencies]
+clap = "=3.2.25"
```

Why bounding this to `"=3.2.25"`?
Can we remove it and control it through `Cargo.lock`?
```suggestion
clap = "3.2.25"
```

---

It should also be optional as the TUI/CLI stuff.